### DISCUSSION TITLE
Budibase API integration with node-red

### DISCUSSION THREAD

**Discussion by MadZhaan** [Budibase/budibase#5840] [General]
Hi all!

 I'm currently self-hosting both Budibase and Node-red locally. I just wanna ask if it is possible to use Budibase API as a data source for Node-red? If it is possible, how do I do it?

**Comment by melohagan** [comment]
Hi @MadZhaan 

I'm not familiar with Node Red, but the Budibase API can be used like any other REST API 

e.g. Retrieve a row: https://docs.budibase.com/reference/get_tables-tableid-rows-rowid

I found this tutorial which may be useful: https://stevesnoderedguide.com/node-red-http-request-node-beginners

**Reply by MadZhaan** [reply]
Thanks @melohagan. That helped me a lot understanding the http request node. I guess I need to do more research on it.

**Reply by melohagan** [reply]
You're welcome, best of luck :)
